python/nav/ipdevpoll/jobs.py

```diff
@@ -81,7 +81,8 @@ class JobHandler(object):
 
     def __init__(self, name, netbox, plugins=None, interval=None):
         self.name = name
-        self.netbox = netbox
+        self.netbox_id = netbox
```

Comment:
Actually, this pretty much breaks ipdevpoll's logging framework, it seems. Logged statements from a job run can no longer be traced back to the netbox the job is working on.

Plugin loggers, model loggers, et.al. will actually inspect the call stack, as they are initialized, to figure out what sysname to insert into their log output. By changing this, the inspection will not find a valid `netbox` variable in the stack to get this name from...